### PR TITLE
Closes #132 #136 #138 #139 Re-vamp of the pharmaverse blog

### LINKED ISSUE #132: Re-skin blog site to be similar to examples or core pharmaverse site

### LINKED ISSUE #136: General Issue: Add pharmaverse hex to the tab icon
### Background Information

image

### LINKED ISSUE #138: General Issue: tidy up "pharmaverse" capitalisation across the blog
### Background Information

The text "pharmaverse" should be in lowercase across the blog to match the casing used in the official [pharmaverse (https://pharmaverse.org/) website.

### LINKED ISSUE #139: Revisit the tags and clean them up.  probably less package specific and more general themes?
Revisit the tags and clean them up.  probably less package specific and more general themes?

_Originally posted by @bms63 in https://github.com/pharmaverse/blog/pull/128#discussion_r1555710535_

### PR DESCRIPTION
**Pull request by manciniedoardo**
Closes #132
Closes #136 
Closes #138
Closes #139

Thank you for your Pull Request! We have developed this task checklist to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your blog post.

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update), and make sure the corresponding issue is linked in the Development section on the right hand side
- [x] Run the script from `CICD.R` line by line to first check the spelling in your post and then to make sure your code is compatible with our code-style. Address any incongruences by following the instructions in the file! 
- [x] Choose (possibly several) `tag(s)` or categories from the current list: `c("metadata", "submission", "qc", "ADaMs", "SDTMs", "community", "conferences", "admiral", "roak", "xportr", "metatools", "metacore")` for your blog post. If you cannot find anything that fits your blog post, add your own tag! We occasionally tidy up all `tags` for consistency.
- [x] Blog post is *short*, *personalized*, *reproducible* and *readable*
- [x] Add a disclaimer at the top of your post, if appropriate (example: **Disclaimer** 
This blog contains opinions that are of the authors alone and do not necessarily reflect the strategy of their respective organizations.)
- [x] Address all merge conflicts and resolve appropriately
- [x] Assign two of us (@bms63, @manciniedoardo, @StefanThoma, @kaz462) as reviewers in the PR.
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by manciniedoardo** [comment]
Hi all, I did a bunch of updates in this PR to re-vamp our blog:
-  I made a hex for the pharmaverseblog and saved it in the media folder. This hex now appears in the tab image, in our README and at the top of the blog header 
image
image
- I changed the header colours to match the pharmaverse palette. I removed dark mode as I couldn't get it to work and it looked pretty bad anyway.
- I made the (non)capitalisation of "pharmaverse" consistent across all posts
- I slimmed down the list of categories to the following, and recategorised all posts:
  - TLGs
  - ADaM
  - SDTM
  - Shiny
  - Community
  - Conferences
  - Submissions
  - Metadata
  - Technical
- I resized any images that were too big for the blog preview so they all look nicer now
- I wrote descriptions so that every post now has one.

**Comment by bms63** [comment]
This image is missing 
image

File: R/create_blogpost.R [hunk 1]
```diff
@@ -7,11 +7,7 @@ source("R/help_create_blogpost.R")
 ## The available images are at the moment:
 list.files("media") %>% tools::file_path_sans_ext()
 ## Please choose (possibly multiple) from this tags list:
-c(
-  "metadata", "submission", "qc", "ADaMs", "SDTMs", "community",
-  "conferences", "admiral", "roak", "xportr", "metatools", "metacore",
-  "displays", "falcon"
-)
+c("Metadata", "SDTM", "ADaM", "TLG", "Shiny", "Community", "Conferences", "Submissions", "Technical")
```
Comment (bms63) [inline R/create_blogpost.R:10]: should get updated in the PR template as well

File: index.qmd [hunk 2]
```diff
@@ -12,9 +12,9 @@ listing:
   sort-ui: true
   filter-ui: true
 page-layout: full
-margin-header: Get notified of new posts by joining [pharmaverse/slack](https://pharmaverse.slack.com) Be sure to check out [https://www.r-bloggers.com]( https://www.r-bloggers.com) 
+margin-header: Get notified of new posts by signing up to our periodic newsletter (link upcoming, stay tuned!) or joining [pharmaverse/slack](https://pharmaverse.slack.com). Be sure to check out [https://www.r-bloggers.com]( https://www.r-bloggers.com) as well.
```
Comment (bms63) [inline index.qmd:15]:
```suggestion
margin-header: Get notified of new posts by signing up to our periodic newsletter (link upcoming, stay tuned!) or by joining [pharmaverse/slack](https://pharmaverse.slack.com). Be sure to check out [https://www.r-bloggers.com]( https://www.r-bloggers.com) as well.
```

File: inst/WORDLIST.txt
Comment (bms63) [inline inst/WORDLIST.txt:1]: We should remove these two words from WORDLIST and then the spelling check will fail 
image

**Review by bms63 — CHANGES_REQUESTED** [review]
Looking good!  I am wondering if there is some way we could programmatically enforce the size of the image in our `CICD.R` file.  @StefanThoma any ideas?

**Comment by bms63** [comment]
Such a nice touch!! 
image

**Comment by manciniedoardo** [comment]
> This image is missing image

Yeah I'm having some issues getting it to display... thought it maybe was just on my side, any ideas?

File: R/create_blogpost.R (hunk 1 above)
Comment (manciniedoardo) [reply R/create_blogpost.R:10]: done thanks

**Comment by bms63** [comment]
> > This image is missing image
> 
> Yeah I'm having some issues getting it to display... thought it maybe was just on my side, any ideas?

it is missing the contributiors.png file in the post

**Comment by manciniedoardo** [comment]
> > > This image is missing image
> > 
> > 
> > Yeah I'm having some issues getting it to display... thought it maybe was just on my side, any ideas?
> 
> it is missing the contributiors.png file in the post

can you see it now? because even though i added it, it's still not rendering for me (same with some of the pharmaverse posts, see image below)
image

**Comment by sadchla-codes** [comment]
@manciniedoardo this is really Nice!! I think I might have found a few more places where pharmaverse is in propcase, should they also be in all lowercase 
image

I have also found cases where pharmaverse is all capitalized but this might be a bit tricky since some times the upper case pharmaverse is part of the title for that talk the author referred to and other times it's not. Should the instances where it's the author referring to pharmaverse also be consistent (all lower case) 
image

**Comment by manciniedoardo** [comment]
> Thanks, looks like I missed some, although some of the ones on there are part of capitalised titles or the first word in a sentence, so I left them alone.

I have now addressed the others that needed changing.

**Review by sadchla-codes — APPROVED** [review]
This looks great to me!

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -4,7 +4,8 @@ Please check off each taskbox as an acknowledgment that you completed the task o
...
 - [ ] Run the script from `CICD.R` line by line to first check the spelling in your post and then to make sure your code is compatible with our code-style. Address any incongruences by following the instructions in the file! 
-- [ ] Choose (possibly several) `tag(s)` or categories from the current list: `c("metadata", "submission", "qc", "ADaMs", "SDTMs", "community", "conferences", "admiral", "roak", "xportr", "metatools", "metacore")` for your blog post. If you cannot find anything that fits your blog post, add your own tag! We occasionally tidy up all `tags` for consistency.
+- [ ] Choose (possibly several) `tag(s)` or categories from the current list: `c("Metadata", "SDTM", "ADaM", "TLG", "Shiny", "Community", "Conferences", "Submissions", "Technical")` for your blog post. If you cannot find anything that fits your blog post, add your own tag! We occasionally tidy up all `tags` for consistency.
+- [ ] Add a short description for your blog post in the `description` field at the top of the markdown document.
```
Comment (bms63) [inline .github/pull_request_template.md:8]: Nice addition!!

**Review by bms63 — APPROVED** [review]
i love this hex!

**Pull request merged**